contrib/opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -603,6 +650,90 @@ def _find_timecode_track_start(track):
     )
 
 
+def _find_mastermob_for_sourceclip(aaf_sourceclip):
+    """
+    For a given soure clip, find the related masterMob
+    """
+
+    # If the mobId of the sourceclip is a mastermob, just return that, we are done.
+    if isinstance(aaf_sourceclip.mob, aaf2.mobs.MasterMob):
+        return aaf_sourceclip.mob, None
```

Comment:
What are the two return values of the function?  should probably note that in the function docstring

Reply:
Yes, documenting the 2nd return value would be helpful. As it is, you have to drill down to the end of the next function to see what that is meant to be.